src/main/webapp/app/overview/course-conversations/layout/conversation-messages/conversation-messages.component.html

```diff
@@ -67,7 +67,17 @@
                 <div
                     id="scrollableDiv"
                     #container
-                    [ngClass]="{ 'posting-infinite-scroll-container': posts.length !== 0, 'content-height-dev': contentHeightDev, 'is-fetching-posts': isFetchingPosts }"
+                    [ngClass]="{
+                        'posting-infinite-scroll-container': posts.length !== 0,
+                        'content-height-dev': contentHeightDev,
+                        'is-fetching-posts': isFetchingPosts,
+                        'hide-input-full':
+                            getAsChannel(_activeConversation)?.isAnnouncementChannel &&
+                            _activeConversation !== undefined &&
```

Comment:
we should try to prevent method calls in the template (except for components using signals/ OnPush ChangeDetection)

Reply:
🙏🏼